homeassistant/components/renson/binary_sensor.py

```diff
@@ -0,0 +1,133 @@
+"""Binary sensors for renson."""
+from dataclasses import dataclass
```

Comment:
I suggest to import `annotation` from `__future__`, then you get more freedom with order of declarations.
```suggestion
"""Binary sensors for renson."""
from __future__ import annotations

from dataclasses import dataclass
```